app/controllers/microservices_engine/v1/data_controller.rb

```diff
@@ -34,18 +34,25 @@ def update
         data = params['content']
         if data.present?
           data.each do |endpoint|
-            existing = Connection.where(object: endpoint['object'])
+            existing = Connection.where(object: endpoint['object']).first
```

Comment:
'existing' is a weird variable name. It's confusing too because exists is a method in Rails.